ruby/lint/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Lint - rubocop'
+description: 'Lints using rubocop'
+
+inputs:
+  CONTINUE_ON_ERROR:
+    description: whether to continue on error from linter
+    required: true
```

Comment:
In order to still upload the file so that Sonarscan can pull it down we need to know whether we can exit early and fail up to the calling workflow or upload the lint artifact. If the calling workflow sets `continue-on-error` on this step instead of passing something down we would fail on the Sonarqube-scan step.

I can certainly set it to not required and true by default though. 👍